apps/settings/style/settings.css

```diff
@@ -286,12 +286,10 @@ body[data-filter-by="connectivity"] #root > div [data-category="connectivity"] {
 .disabled h2,
 .disabled label,
 .disabled select,
-.disabled gaia-checkbox,
-.disabled gaia-radio,
-.disabled gaia-switch,
+gaia-checkbox[disabled],
 gaia-radio[disabled],
 gaia-switch[disabled] {
-  opacity: 0.6;
+  opacity: 0.3;
```

Comment:
Sorry for lack communication. In https://bugzilla.mozilla.org/show_bug.cgi?id=1206927#c6 UI designer explicit ask for 30% opacity for disable state